repo/packages/K/kubernetes/100/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "1.0.1-1.9.4-rc1"
```

Comment:
Shouldn't this read `1.0.0-1.9.3` (meaning, we can upgrade from `1.0.0-1.9.3`)? Again, just double checking. Also, this version isn't prefixed with `v`.

Reply:
Yes, i believe that should be the case `1.0.0-1.9.3`